src/dcm/agent/tests/unit/test_configure.py

```diff
@@ -361,3 +361,33 @@ def check_version(conf):
         conf = config.AgentConfig([os.path.join(
             self.test_base_path, "etc/agent.conf")])
         self.assertTrue(conf.intrusion_detection_ossec)
+        self.assertEqual(conf.intrusion_detection_alert_threshold, 10)
+
+    def test_configure_ossec_alert_threshold_change(self):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename method "test_configure_ossec_alert_threshold_change" to match the regular expression ^[a-z_][a-z0-9_]{2,30}$. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.int.enstratius.com/coding_rules#rule_key=python%3AS100)